semantic-ui-react-kitchensink/assets/index.html

```diff
@@ -3,12 +3,12 @@
 <head>
     <meta charset="UTF-8">
     <title>Semantic-ui-react demo</title>
-    <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/semantic-ui@2.4.1/dist/semantic.min.css"
+    <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/semantic.min.css"
```

Comment:
can we pull the styling from node_modules instead?

Reply:
You mean using https://www.npmjs.com/package/semantic-ui-css?

Reply:
> can we pull the styling from node_modules instead?

Not sure how this is done.